kubech

```diff
@@ -26,6 +26,15 @@ EOF
 }
 
 
+#
```

Comment:
Please update the readme file since your change fixes the note mentioned there.

It's [line no. 94](https://github.com/aabouzaid/kubech/blame/master/README.md#L94) in the read me file.